### PR TITLE
Add HeatmapVis story

### PR DESCRIPTION
**Pull request by axelboc**
image

### CODE REVIEW

File: src/stories/HeatmapVis.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const Template: Story<HeatmapVisProps> = (args): ReactElement => (
+  <HeatmapVis {...args} />
+);
+
+export const Default = Template.bind({});
+
+Default.args = {
+  dataArray: ndarray<number>(
+    dataset1.value.flat(Infinity) as number[],
+    dataset1.shape.dims.slice(0).reverse()
```
Comment (axelboc) [inline src/stories/HeatmapVis.stories.tsx:21]: Just transposing the dataset for a better visualization.

Comment (loichuder) [reply src/stories/HeatmapVis.stories.tsx:21]: Did you try [transpose](https://github.com/scijs/ndarray#arraytransposep0-p1-) instead ?

Anyway, I think the comment has its place in the code as well.

File: .storybook/decorators/FillHeight.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+function FillHeight(VisCanvasStory: Story) {
+  return (
+    <div style={{ display: 'flex', height: '100vh' }}>
+      <VisCanvasStory />
+    </div>
+  );
+}
+
+export default FillHeight;
```
Comment (loichuder) [inline .storybook/decorators/FillHeight.tsx:12]: Nice factorization :+1:

**Review by loichuder — APPROVED** [review]
:sparkles:

**Pull request merged**